faq/README.md

```diff
@@ -0,0 +1,143 @@
+# Freedeck FAQ
+
+## Ordering stage
+
+### What components do I need
+
+#### Through hole PCB (Adviced to starters in soldering)
+You will need:
+- PCB / Perfboard
+- Arduino Pro Micro (32u4, 16MHz, 5V)
+- 2x 4051 Multiplexers
+- 6x SSD1306 0.96" OLED
+- 6x 3x4x2mm SMD buttons
+- SPI-SDcard reader (Micro SD form)
+
+#### SMD varient (For advanced solderers)
+See the BOM with the PCB you are using
+
+### Where can I find resources
+You can find resources in the various github repo's, or you can ask us in the [discord](https://discord.gg/sEt2Rrd)
+
+### Where can I order the PCB
+We do have a good experience with [JLCPCB](https://jlcpcb.com) (Not affilated). You can also find some other company like EuroCurcuits
+
+### What do i need to order the PCB
+The GERBER-file zip that you can find in the hardware github repo
```

Comment:
maybe add a direct link to the file here

Reply:
Then we also need to keep it up to date. But it has been added!